lib/runBlock.js

```diff
@@ -83,7 +83,7 @@ module.exports = function (opts, cb) {
     async.eachSeries(block.transactions, processTx, cb)
 
     function processTx (tx, cb) {
-      var gasLimitIsHigherThanBlock = new BN(block.header.gasLimit).cmp(new BN(tx.gasLimit).add(gasUsed)) === -1
+      var gasLimitIsHigherThanBlock = new BN(block.header.gasLimit).lt(new BN(tx.gasLimit).add(gasUsed))
```

Comment:
Not worth for the switch I suppose. But I think this is a potential consensus issue here.

Just looked it up in the white paper:

![bildschirmfoto 2017-08-16 um 12 53 16](https://user-images.githubusercontent.com/931137/29360155-ef69c71a-8281-11e7-88be-1bc1f4b04a06.png)

So it's really only invalid in ``gt`` case while we have (implicitly) used ``gte`` for the error case. So the ``equal`` case is falsely included there.